packages/cli/src/commands/fix/command.ts

```diff
@@ -61,46 +62,23 @@ fixCommand
       .default(process.cwd())
       .hideHelp()
   )
-  .action(async (targetPath: string, options: FixCommandOptions) => {
-    await fix(targetPath, options);
+  .action(async (srcPath: string, options: FixCommandOptions) => {
```

Comment:
could you please use targetSrcPath... just trying to make it clearer down the code.